geocatbridge/servers/views/geonetwork.ui

```diff
@@ -115,6 +115,57 @@
           </property>
          </widget>
         </item>
+        <item row="4" column="0">
+         <widget class="QLabel" name="label_33">
+          <property name="text">
+           <string>Group</string>
+          </property>
+         </widget>
+        </item>
+        <item row="4" column="1">
+         <widget class="QLineEdit" name="txtGeonetworkGroup">
+          <property name="text">
+           <string/>
+          </property>
+          <property name="placeholderText">
+           <string>Group Id</string>
+          </property>
+         </widget>
+        </item>
+        <item row="5" column="0">
+         <widget class="QLabel" name="label_35">
+          <property name="text">
+           <string>Assign to current Catalogue</string>
+          </property>
+         </widget>
+        </item>
+        <item row="5" column="1">
+         <widget class="QLineEdit" name="txtGeonetworkAssignToCat">
+          <property name="text">
+           <string/>
+          </property>
+          <property name="placeholderText">
+           <string/>
+          </property>
+         </widget>
+        </item>
+        <item row="6" column="0">
+         <widget class="QLabel" name="label_37">
+          <property name="text">
+           <string>Transform with</string>
+          </property>
+         </widget>
+        </item>
+        <item row="6" column="1">
+         <widget class="QLineEdit" name="txtGeonetworkTransformWith">
```

Comment:
I am thinking of adding a dropdown here with all the available options for a more user-friendly experience. But this will do for now.

Reply:
I like the idea of a dropdown